.travis.yml

```diff
@@ -0,0 +1,45 @@
+language: python
+cache: pip
+
+env:
+  global:
+    - PYTHONPATH="./tests/"
+    - DJANGO_SETTINGS_MODULE="settings_test"
+
+matrix:
+  include:
+    - python: 3.6
+    # Workaround for Python 3.7 support. See:
+    # https://github.com/travis-ci/travis-ci/issues/9815
+    - python: 3.7
+      dist: xenial
+      sudo: true
+
+# Workaround for testing Boto3 on Travis. Taken from:
+# https://github.com/azavea/django-amazon-ses/blob/57ed8619435194742308c6f15cbf9c91b231b88a/.travis.yml#L17-L19
+before_install: sudo rm /etc/boto.cfg
+
+install: pip install .[tests]
+
+script: django-admin test --noinput
```

Comment:
`scripts/test` and `scripts/update` use a virtualenv-oriented approach that doesn't work on Travis (which already sets up a virtualenv for each Python build). I don't anticipate setup or testing instructions will change at all, so I decided to follow the easiest path and duplicate the commands here.

Two alternative approaches spring to mind:

1. Add conditional checks to `scripts/test` and `scripts/update` to run separate commands if the scripts are running on CI
2. Refactor the dev environment to use containers and update Travis to eliminate the prefab Python environment

I decided against 1) because it felt equally duplicative and 2) because my experience containerizing Python tests for Travis in https://github.com/azavea/grout introduced much slower CI builds and a bunch of extra development artifacts.